app/controllers/sales_controller.rb

```diff
@@ -0,0 +1,48 @@
+require 'date'
+
+class SalesController < ApplicationController
+
+  def show
+    @sale = Sale.where(vendor_id: params[:id]).order(purchase_time: :asc)
+    @group = Sale.group("strftime('%m', purchase_time)").count
```

Comment:
I'm currently pretty unsure of when methods should go in the controller vs. the model. CRUD stuff in Controller, other stuff in the model? Or is their more nuance than that?